### DISCUSSION TITLE
Request to upgrade buildx in Docker Desktop

### DISCUSSION THREAD

**Discussion by AdilKh4n** [docker/buildx#2752] [Q&A] [answered]
Where can I put a request to include the latest stable buildx version (v0.17.1) in Docker Desktop for MacOS?

**Comment by thompson-shaun** [comment, marked as answer]
👋 @AdilKh4n 

The next release of Docker Desktop for MacOS (4.35.0) will have buildx v0.17.1 in it

See the [release notes](https://docs.docker.com/desktop/release-notes/#4350) for more information.

(Thanks for using the Q&A/discussion)

**Comment by AdilKh4n** [comment]
@thompson-shaun Thank you for your response. Do you know when is the next release scheduled?

**Reply by thompson-shaun** [reply]
Great question! That would be today (10/24) -- see https://docs.docker.com/desktop/release-notes/#4350 for more information.
